projects/ngx-charts-on-fhir/src/lib/fhir-chart-legend/fhir-chart-legend.module.ts

```diff
@@ -4,9 +4,10 @@ import { FhirChartLegendComponent } from './fhir-chart-legend.component';
 import { MatIconModule } from '@angular/material/icon';
 import { OverlayModule } from '@angular/cdk/overlay';
 import { MatButtonModule } from '@angular/material/button';
+import { SortDatasets } from './sort-by.pipe';
```

Comment:
filename is still using the old pipe name
```suggestion
import { SortDatasets } from './sort-datasets.pipe';
```

Reply:
Okay, @semicolin I will update this.